authentik/core/api/applications.py

```diff
@@ -216,6 +228,8 @@ def list(self, request: Request) -> Response:
         if superuser_full_list and request.user.is_superuser:
             return super().list(request)
 
+        only_with_launch_url = str(request.query_params.get("only_with_launch_url", "false")).lower()
```

Comment:
I still there has to be a better way to parse this value than doing manual string parsing...but I suppose not part of this PR..I know we can create serializers and pass `request.query_params` into that and use that but that's a whole lot more code